k8s/config.yaml

```diff
@@ -0,0 +1,26 @@
+# ConfigMap for sparc-dev
```

Comment:
I would either change this to "ConfigMap for sparc" or delete the first two lines of this file.